src/python/tests/appengine/libs/request_cache_test.py

```diff
@@ -53,8 +53,7 @@ class CacheTest(unittest.TestCase):
 
   def setUp(self):
     test_helpers.patch(self, [
-        'libs.auth.get_current_request',
-        'libs.auth.get_cache_backing'
+        'libs.auth.get_current_request', 'libs.request_cache.get_cache_backing'
```

Comment:
do you still need to mock get_current_request?

Reply:
no, we don't need it. Removed in latest commit